src/CMakeLists.txt

```diff
@@ -142,6 +142,7 @@ set( MIOpen_Source
     graphapi/rng.cpp
     graphapi/tensor.cpp
     graphapi/variant_pack.cpp
+    graphapi/conv_bias_res_add_activ_forward_executor.cpp
```

Comment:
Please keep filenames sorted alphabetically. Although I don't know the convention we use for underscore, since it's ascii code falls between the capitals and lowercase letters :)

Reply:
agree. Sorting should be maintained.

Reply:
Does anybody know any linter for CMAKE?